pkgs/servers/http/nginx/modules.nix

```diff
@@ -462,6 +484,17 @@ in
     inputs = [ pkgs.ffmpeg_3 ];
   };
 
+  vod = {
+    src = fetchFromGitHub {
+      name = "vod";
+      owner = "kaltura";
+      repo = "nginx-vod-module";
+      rev = "e46079f51282d5a378e6911714b5f3a533bb7700";
```

Comment:
They actually seem to be doing releases, why not stick to that?

Reply:
Unfortunately they didn't release one with recent bugfixes such as https://github.com/kaltura/nginx-vod-module/commit/a5c6825c9ae595d81c143515d3c02e2202abaebe.